### ISSUE TITLE
image detail lose in general imaging chip

### ISSUE THREAD

**Issue by zhanzhaokai** [cchen156/Learning-to-See-in-the-Dark#57]
Hi, cchen156. Thank you for your contribution! we  train and test SID in our own DataSet（raw data from general imaging chip）. we find serious detail lose phenomenon especially for text and grass. we tried different CNN structure.

**Comment by cchen156** [comment]
This is normally because the input data is not good. Can you check your raw in Photoshop? Can you see the text and grass in Photoshop? You could adjust the exposure in Photoshop to make the content visible.

**Comment by zkvz** [comment]
Yes, we check our  raw data in Photoshop. we find chip's ISP can get more details with little noise and CNN get less details with no noise (about  0.01 lux), CNN get a oversmoothed result. we find the same  phenomenon in sony dataset. When the image signal strength is weak, CNN maybe smooth out the details. I don't have any good solution, could you give me some advice.Thank you !
